### PR TITLE
chore: wait for post to finish instead of waiting a fixed time

### PR DESCRIPTION
**Pull request by gardleopard**
This will hopefully make the test faster and more stable

### CODE REVIEW

**Review by gastonfournier — APPROVED** [review]
Lovely!

File: frontend/cypress/integration/import/import.spec.ts [hunk 1]
```diff
@@ -113,8 +117,7 @@ describe('imports', () => {
         cy.get("[data-testid='IMPORT_CONFIGURATION_BUTTON']").click();
         // cy.contains('Import completed');
```
Comment (kwasniew) [inline frontend/cypress/integration/import/import.spec.ts:118]: I'm wondering why this commented out code was commented out. Ideally UI tests should wait on the UI changes not HTTP API interception

Comment (kwasniew) [reply frontend/cypress/integration/import/import.spec.ts:118]: However if your change fixes the issue then thumbs up

Comment (gardleopard) [reply frontend/cypress/integration/import/import.spec.ts:118]: I dont know

Comment (gardleopard) [reply frontend/cypress/integration/import/import.spec.ts]: Added this back now @kwasniew, good spot. Now we wait for the POST to finish and then validate ui.

**Pull request merged**